README.md

```diff
@@ -0,0 +1,41 @@
+## Archives Space End-to-End Tests
+
+#### How to install
+
+Clone this repository on your machine, navigate to the root application folder, and run:
+
+```
+bundle install
+```
+
+### How to run the tests
+
+#### Run on remote host
+To run the tests on the `https://e2e.archivesspace.org`, run:
```

Comment:
Not really a code question, but if this is going to be open, have we put any thought into how we might manage if members of the community begin running tests against it? Do we need to have it protected by some sort of key (or maybe it is already?)

Reply:
The setup on [https://e2e.archivesspace.org](https://e2e.archivesspace.org) closely resembles those on [https://test.archivesspace.org/](https://test.archivesspace.org/) and [https://sandbox.archivesspace.org/](https://sandbox.archivesspace.org/), so I personally see no reason that it should be restricted, at least for the moment. It is totally fine if anybody runs any amount of tests on it, as they are isolated. Besides that, the tests on the workflow run on a Docker container when we push changes, which is going to be the ordinary workflow down the line. There is also a chance that we may decide to abolish the instance on [https://e2e.archivesspace.org](https://e2e.archivesspace.org) in the future, and strictly rely on the docker container.

Reply:
Ok!